modules/caddytls/cipher_suites.go

```diff
@@ -0,0 +1,107 @@
+// Derived from https://github.com/golang/go/commit/0ee22d9 which should be
+// included in Go 1.14.  Once Go 1.14 is released, we can remove this file.
```

Comment:
Go 1.14 is in RC, and I think we won't get around to merging this until Go 1.14 is released, probably within the next couple weeks. So, just keep in mind, I think we will be removing this file before the merge.

Reply:
:+1: I'm in no rush.  Happy to wait for Go 1.14 to land.